Die Zeit.js

```diff
@@ -169,32 +144,40 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "newspaperArticle",
-				"creators": [],
-				"notes": [],
-				"tags": [
-					"Muammar al-Gaddafi",
-					"Muammar al-Gaddafi",
-					"Mustafa Abdel Dschalil",
-					"Stadt",
-					"Bani Walid",
-					"Berg"
+				"title": "Libyen: Rebellen bereiten Angriff auf Bani Walid vor",
+				"creators": [
+					{
+						"firstName": "",
+						"lastName": "AFP",
+						"creatorType": "author"
```

Comment:
On the webpage they have even an <a href="http://www.zeit.de/autoren/afp">author page for AFP</a> an other press agencies. There is no HTML markup which indicates here anything. We can test for `firstName==""`, but then I would suggest to go for the fieldMode solution. Okay?